### PR TITLE
Correct sensuctl command examples

### LINKED ISSUE #2014: sensuctl command examples don't work
The examples given for using `sensuctl command` don't work as provided.

Additionally, some clarification is needed.  Is using sensu-email-handler a good example in the first place?  If the command being ran is mapped to the bin/entrypoint for the asset then that file doesn't even exist in that asset.  As well, it's meant as a pipe handler, expecting to read from STDIN, can you do that with `sensuctl command exec`?
## This is an issue with:

- [ ] Bug (site functionality or styling)
- [X] Errata (fix needed for doc content)
- [ ] New content (guide wanted)
- [ ] Update (Add missing or refresh existing content)
- [ ] Enhancement (add new site functionality)

## Current Behavior

Copy/Paste of the provided example of sensu-email-handler doesn't work.  It does appear to get further without the parentheses.  The failure after that may or may not be an actual issue with Bonsai integration.
```
[vagrant@backend ~]$ sensuctl command install sensu-email-handler (sensu/sensu-email-handler:0.2.0)
-bash: syntax error near unexpected token `('
[vagrant@backend ~]$ sensuctl command install sensu-email-handler sensu/sensu-email-handler:0.2.0
fetching bonsai asset: sensu/sensu-email-handler:0.2.0
Error: one or more asset builds are required
```
## Affected Docs Pages

[Extend sensuctl with commands](https://docs.sensu.io/sensu-go/5.15/sensuctl/reference/#extend-sensuctl-with-commands)
## Steps to Reproduce (for bugs)

1.
2.
3.

## Context

* Browser(s) and version(s): 
* OS:

### PR DESCRIPTION
**Pull request by hillaryfraley**
## Description
Remove email handler examples from https://docs.sensu.io/sensu-go/latest/sensuctl/reference/#extend-sensuctl-with-commands

## Motivation and Context
Closes https://github.com/sensu/sensu-docs/issues/2014

## Review Instructions

- As is, this PR only removes the incorrect examples.
- I don't think this completely resolves #2014 -- specifically, I think I need to explain (briefly) what kinds of assets will work with `sensuctl command` and why. Jef's comment in the issue, "The sensuctl command concept requires a specially packaged asset with a specially named binary. It is not a general command runner... handler assets won't work." seems like a good starting point.
- I think we should replace the examples with examples that will work, but Jef's comment "I don't think we have an existing example of a sensuctl command asset in bonsai yet, which makes getting the documentation for bonsai integration working sort of a chicken and egg problem." sounds like it could be challenging to find right now (I may be misunderstanding).

Rather than review, I think I need to pair with someone on this to get it over the finish line.

### CODE REVIEW

**Comment by hillaryfraley** [comment]
ec2 plugin needs to be added to Bonsai -- when added, use as the example for `sensuctl command`

Hillary:

- Ask Eng whether ec2 plugin will be posted soon (get timeline).
- If this will not happen this week, merge this PR to remove details that are inaccurate.

https://github.com/calebhailey/sensu-ec2-discovery

update: EC2 plugin should be published this week. Hold PR and update with EC2 info when published.

**Comment by hillaryfraley** [comment]
Hillary:

- Ask Eng whether ec2 plugin will be posted soon (get timeline).
- If this will not happen this week, merge this PR to remove details that are inaccurate.

https://github.com/calebhailey/sensu-ec2-discovery

**Comment by hillaryfraley** [comment]
update: EC2 plugin should be published this week. Hold PR and update with EC2 info when published.

**Comment by hillaryfraley** [comment]
I updated the examples to use the EC2 discovery plugin as the example. This is now ready for review!

**Comment by hillaryfraley** [comment]
I updated the examples to use the EC2 discovery plugin as the example. This is now ready for review!

Review question: I'm not sure the phrasing is correct here. Are we installing a command? Or using `sensuctl command install` to install a plugin?

I'm looking at how Sean wrote the README for EC2 discovery and it looks like the phrasing "using `sensuctl command install` to install a plugin is correct. https://bonsai.sensu.io/assets/portertech/sensu-ec2-discovery

File: content/sensu-go/5.16/sensuctl/reference.md [hunk 1]
```diff
@@ -886,18 +886,33 @@ To install a sensuctl command from Bonsai or a URL:
 sensuctl command install [ALIAS] ([NAMESPACE/NAME]:[VERSION] | --url [ARCHIVE_URL] --checksum [ARCHIVE_CHECKSUM]) [flags]
 {{< /highlight >}}
 
+You can install a command by using the Bonsai asset name or from a URL.
```
Comment (cwjohnston) [inline content/sensu-go/5.16/sensuctl/reference.md]:
```suggestion
To install a command plugin, use the Bonsai asset name or specify a URL and SHA512 checksum.
```

File: content/sensu-go/5.16/sensuctl/reference.md [hunk 2]
```diff
@@ -886,18 +886,33 @@ To install a sensuctl command from Bonsai or a URL:
...
+
+Replace `[flags]` with the flags you want to use.
+Run `sensuctl command install -h` to view flags.
+Flags are optional and apply only to the `install` command &mdash; they are not saved as part of the command you are installing.
+
+To install a command from the [Sensu EC2 Discovery Plugin][37] with no flags:
 
 {{< highlight shell >}}
-sensuctl command install sensu-email-handler (sensu/sensu-email-handler:0.2.0)
+sensuctl command install sensu-ec2-discovery (portertech/sensu-ec2-discovery:0.3.0)
```
Comment (cwjohnston) [inline content/sensu-go/5.16/sensuctl/reference.md]:
```suggestion
sensuctl command install sensu-ec2-discovery portertech/sensu-ec2-discovery:0.3.0
```

I think the parentheses in the `sensuctl command install` usage output are meant to indicate that one should provide either a `[NAMESPACE/NAME]:[VERSION]` or a `url` and `checksum`, meaning those parentheses should not be used in the actual command.

That said, this command isn't yet working for me, with or without the parentheses. I'm getting `Error: namespace cannot be empty` but I'm not sure if this a problem with sensuctl or what bonsai is returning.

**Review by cwjohnston — CHANGES_REQUESTED** [review]

**Comment by hillaryfraley** [comment]
Potential issue: https://sensu.slack.com/archives/C60EEQFH8/p1579038212406600?thread_ts=1578980402.337100&cid=C60EEQFH8

**Comment by hillaryfraley** [comment]
Bug in `sensuctl command install` for 5.16; fixed for 5.17.

Need to replicate these docs into 5.17. Add a note in 5.16 docs about need to update to 5.17 to use sensuctl command plugins.

**Comment by cwjohnston** [comment]
https://github.com/sensu/sensu-go/pull/3504 fixes `sensuctl command install` via Bonsai, and has been merged for the 5.17 release

**Pull request merged**